### PR TITLE
Adding Homeoffice DRT repos

### PR DESCRIPTION
**Pull request by ngu04**
Adding few more DRT home office repos

### CODE REVIEW

**Comment by ngu04** [comment]
Let me know if you need any manual validation

File: repos-github.md [hunk 1]
```diff
@@ -1367,7 +1367,14 @@
 - ua-parser/uap-scala
 - ucb-bar/chisel-testers2
 - UdashFramework/udash-core
+- UKHomeOffice/bluebus
+- UKHomeOffice/drt-api-import
+- UKHomeOffice/drt-analytics
+- UKHomeOffice/drt-birmingham-schema
```
Comment (tgodzik) [inline repos-github.md:1373]: Is it a private repo? I can't find it under https://github.com/UKHomeOffice/drt-birmingham-schema

Comment (ngu04) [reply repos-github.md:1373]: yes , It has be to private . How do we manage it ?

Comment (ngu04) [reply repos-github.md:1373]: I guess I need to remove it from here and run locally or part of our build

**Pull request merged**